UnityWeld/Ioc/IWeldContainerIoc.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using UnityWeld.Binding;
+
+namespace UnityWeld.Ioc
+{
+    public interface IWeldContainerIoC
```

Comment:
Should add a `/// <summary> ...` comment explaining how the interface works so that users of the API can implement it themselves.